src/LSSVMRouter.sol

```diff
@@ -48,7 +48,7 @@ contract LSSVMRouter {
     }
 
     struct RobustPairNFTsFoTokenAndTokenforNFTsTrade {
```

Comment:
`RobustPairNFTsFoTokenAndTokenforNFTsTrade` => `RobustPairNFTsForTokenAndTokenforNFTsTrade`

Reply:
no point in changing it now, will make all contracts interacting with previous versions incompatible.